easy_access/db/update.py

```
@@ -1332,94 +1393,115 @@ async def persist_courses(
+                    for k, v in ud.items():
+                        if not k.startswith("_"):
+                            setattr(course_obj, k, v)
+                    await session.commit()
+            except Exception as exc:  # pragma: no cover
+                logger.error(f"Error updating course {code}: {exc}")
 
     logger.info(f"Successfully persisted {len(courses_data)} courses")
 
 
-async def _apply_course_teacher_relations(course_obj, rel_payload: dict, PersonModel):
+async def _apply_course_teacher_relations(
```

Ruff (0.14.6)

Missing return type annotation for private function `_apply_course_teacher_relations`

Add return type annotation: `None`

(ANN202)